src/components/elements/CardComponent.vue

```diff
@@ -23458,8 +23462,7 @@ export default {
         MultiClass: "#9A7038",
         HQ: "#9A7038"
       }
-
-      if (this.cardType && this.cardType.HQ) {
+      if (this.Type && this.Type.HQ) {
```

Comment:
This is the interesting part.
It fixes the endless warnings in the logs and it also fixes the name of headquarters not being displayed in the right color.